File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/InviteRulesVO.kt
package com.sdkwork.app

data class InviteRulesVO(
    val title: String? = null,
    val description: String? = null,
    val inviterReward: Int? = null,
    val inviteeReward: Int? = null,
    val rules: List<String>? = null
)

File: sdkwork-app-sdk-csharp/Models/InviteRulesVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class InviteRulesVO
    {
        public string? Title { get; set; }
        public string? Description { get; set; }
        public int? InviterReward { get; set; }
        public int? InviteeReward { get; set; }
        public List<string>? Rules { get; set; }
    }
}
